### DISCUSSION TITLE
Why doesn't 9x9 mesh probe the top right point?

### DISCUSSION THREAD

**Discussion by peeter-virk** [mriscoc/Ender3V2S1#646] [Q&A]
On the most recent build, the 9x9 bed mesh the last point won't be probed instead the probing finishes with 
80 points probed.
printer: Ender3v2
board: one with the lowest number supported.
I used the precompiled code.

**Comment by mriscoc** [comment]
https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
https://github.com/mriscoc/Ender3V2S1/wiki/3D-BLTouch#notes-about-crtouch
Probe offsets / HS mode

**Comment by dfinberg** [comment]
Most likely, your nozzle to probe X/Y combined with the mesh insert you have set combined with the physical max for Y don't allow the probe in that position.  Smart Fill will auto generate a value for that position if you use it.

**Comment by TheWebMachine** [comment]
I've had this happen randomly. Running mesh again without changing anything usually succeeds. Not sure of the cause, but it has happened super rarely. Maybe twice out of ~50 times.

I, too, am running precompiled; Ender3S1-F4-UBL-20221222 in my case.

**Reply by mriscoc** [reply]
Turn off HS mode. If that doesn't fix it, check your cables. Put a ferrite ring to avoid electrical interferences.

**Reply by TheWebMachine** [reply]
I am using the CR probe mod to move it to [near] Y-offset = 0 (-1.8mm in the case of the Plus model). I have had it happen in both HS mode and normal mode, but I also have had none of the usual issues with HS mode on my Plus model. 

I hadn't considered the possibility of eletrical interference, though. Quite possible since my probe wires have a lot more unfettered slack with the mod backplate and new probe position. I'll throw a ferrite bead on the line and keep an eye out for it to happen again. Who knows cuz I've only ever had it happen twice so far.
